Mathlib/LinearAlgebra/BilinearForm/Hom.lean

```diff
@@ -6,6 +6,7 @@ Authors: Andreas Swerdlow, Kexing Ying
 import Mathlib.LinearAlgebra.BilinearMap
 import Mathlib.LinearAlgebra.BilinearForm.Basic
 import Mathlib.LinearAlgebra.Basis
+import Mathlib.Algebra.Algebra.Bilinear
 
 /-!
 # Bilinear form and linear maps
```

Comment:
Can you add a TODO comment in this docstring that indicates a lot of the file is now pointless (for instance, `toLinHom` is now the identity!).

I don't think we need to fix that in this PR, but it would be good to record that the file needs further work to avoid confused readers.

Reply:
Done.